index.html

```diff
@@ -2,25 +2,30 @@
 <html>
 <head>
   <meta charset="utf-8" />
-  <title>Black Screen</title>
-  <link rel="stylesheet" type="text/css" href="compiled/all.css" media="screen"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title></title>
+  
+  <!-- TOOD: Merge and minimify -->
+  <link rel="stylesheet" type="text/css" href="dist/main.css" media="screen"/>
   <link rel="stylesheet" type="text/css" href="bower_components/filament-sticky/fixedsticky.css" media="screen"/>
+  <!-- /.TODO -->
 
   <script>
     window.jQuery = window.$ = require('jquery');
   </script>
 
   <script src="bower_components/livereload-js/dist/livereload.js?host=localhost"></script>
+  
+  <!-- TODO: Merge and minimify -->
   <script src="bower_components/caret.js/dist/jquery.caret.min.js"></script>
   <script src="bower_components/filament-sticky/fixedsticky.js"></script>
-  <script src="compiled/views/main.js"></script>
-  <script src="decorators/json.js"></script>
+  <!-- /.TODO -->
+  
+  <script src="dist/scripts/views/main.js"></script>
 </head>
-<body>
-
-<div id="sizes-calculation">1</div>
-
-<div id="black-board"></div>
+<body dir="ltr">
```

Comment:
Sorry, this is not necessary I will remove it.

``dir="ltr"`` = Direction of the text: left to right